Reqnroll/TestThreadContainerInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Reqnroll.BoDi;
+
+namespace Reqnroll;
+
+internal class TestThreadContainerInfo
```

Comment:
I would move this class to the `Infrastructure` folder and namespace. I would try to minimize the usage of the top level namespace (although we were not fully consistent in this earlier).

Reply:
No problem. I moved it.